README.md

```diff
@@ -1492,22 +1493,17 @@ _Libraries for programming devices of the IoT._
 ## Job Scheduler
 
 _Libraries for scheduling jobs._
-- [Cadence-client](https://github.com/uber-go/cadence-client) - A framework for authoring workflows and activities running on top of the Cadence orchestration engine made by Uber.
```

Comment:
--An action needed on this

Reply:
would you give us more context on what action are you referring to!